dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/OAuthFlowsValidator.java

```diff
@@ -34,6 +37,45 @@ private OAuthFlowsValidator() {}
     /** {@inheritDoc} */
     @Override
     public void validate(ValidationHelper helper, Context context, String key, OAuthFlows t) {
-        // TODO Auto-generated method stub
+        if (t.getImplicit() != null) {
+            OAuthFlow implicit = t.getImplicit();
+            String type = "implicit OAuthFlow";
+            if (StringUtils.isNotBlank(implicit.getTokenUrl())) {
+                final String message = Tr.formatMessage(tc, "nonApplicableField", "tokenUrl", implicit.getTokenUrl(), type);
```

Comment:
nonApplicableField message takes 3 param values. Only 2 are mentioned here. The order needs to match the fields in the message as well.

Reply:
Right. missed it.